src/main/java/org/folio/rest/repository/FeeFineActionRepository.java

```diff
@@ -132,26 +132,59 @@ public FeeFineActionRepository(Map<String, String> headers, Context context) {
   }
 
   public Future<List<Feefineaction>> findActionsByTypeForPeriodAndOwners(Action typeAction,
-    String startDate, String endDate, List<String> ownerIds, int limit) {
+                                                                         String startDate, String endDate, List<String> ownerIds, int limit) {
 
     String ownerIdsFilter = buildOwnerIdsFilter(ownerIds);
+    String selectWithJoin = "SELECT actions.jsonb FROM %1$s.%2$s actions " +
+      "LEFT OUTER JOIN %1$s.%3$s accounts ON actions.jsonb->>'accountId' = accounts.jsonb->>'id' ";
+    String orderBy = "ORDER BY actions.jsonb->>'dateAction' ASC ";
+    String query = null;
+    Tuple params = null;
+
+    if (startDate != null && endDate != null) {
```

Comment:
Can we have a List<String> of conditions like "actions.jsonb->>'dateAction' >= $4", "actions.jsonb->>'dateAction' < $5" and join it with " AND "?